daktari/config.py

```diff
@@ -119,6 +148,17 @@ def check_version_compatibility(config_path: Path, raw_config: str) -> bool:
         print(red(f"❌  Invalid daktari_version in {config_path}: {required_version}"))
         return False
 
+    if not is_python_version_on_path():
+        print(
+            yellow(
+                "⚠️  A Python version was detected on your PATH. "
+                "However the version on your PATH does not match the current version of Python 3 installed. "
+                "It is possible Daktari is/will be installed under a directory not on your PATH. "
+                "You may need to update your PATH to use the currently install Python version. "
+                "E.g. /Users/username/Library/Python/3.11/bin"
```

Comment:
This is a Mac specific path, so are Linux users likely to see it? I guess not because they'll probably have `python3` in `/usr/bin`?